lib/setup.js

```diff
@@ -65,7 +65,9 @@ function registerPartials(items) {
   // Compile all the loaded partials
   _.forIn(hbs.partials, function (value, key) {
     if (_.isString(value)) {
-      hbs.partials[key] = hbs.compile(value);
+      //preventIndent:true solve issue with  <pre> tag/white-space: pre
+      // see http://handlebarsjs.com/reference.html compile() options 
+      hbs.partials[key] = hbs.compile(value, { preventIndent:true });
```

Comment:
Add another whitespace in here
```suggestion
      hbs.partials[key] = hbs.compile(value, { preventIndent: true });
```

Reply:
done